app/models/ems_refresh/save_inventory.rb

```diff
@@ -115,7 +115,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        h[:_object] = found
+        h[:_object] = found.reload
```

Comment:
we do so much logic that is aleviated by `h[:_object]`

we need to revisit inventory code that creates hashes and other type lookup entries to cross index entries.